docs/source/user/installation/install-linux.md

```diff
@@ -0,0 +1,39 @@
+# GNU + Linux Installation Guide
```

Comment:
Rename this file to `installation/install-gnu+linux.md` please

Reply:
https://github.com/googlefonts/fontbakery/pull/2158/commits/de08c60ac5c2749542ebf7e04d162caa8826254d